### DISCUSSION TITLE
How to setup the vscode extension

### DISCUSSION THREAD

**Discussion by enricoschaaf** [quarylabs/sqruff#933] [Q&A]
Hi 👋🏽 Thanks for this project! 

I am trying to setup the vscode extension, I have sqruff installed, the extension installed and set the extension as the default formatter for sql. Sadly when trying to format postgres via the explicit format command vscode says Extension 'sqruff' is configured as formatter but it cannot format 'MS SQL'-files. Sadly it seems there are no other sql languages available to change it to.

Could you tell me what I need to change to make it work? 😊

**Comment by benfdking** [comment]
Hey @enricoschaaf, sorry we missed your message. Have you looked at making a config file?

**Reply by enricoschaaf** [reply]
Hi 👋 Thanks for the reply. I have both a score config and a sqruff config, the issue is vscode does not recognize the sqruff extension as a proper formatter and because of that I cannot use it for auto formatting.

**Reply by benfdking** [reply]
Hey @enricoschaaf,

Sorry about this. Would you be able to document your setup in an issue? I think I see what you are saying but it would be nice to test it E2E

**Reply by enricoschaaf** [reply]
Sure

**Reply by benfdking** [reply]
Thanks!
